src/components/table/tableContainer/__tests__/tableContainer.test.tsx

```diff
@@ -0,0 +1,47 @@
+import { render } from '@testing-library/react'
+import { TableContainer } from '../index'
+import { TableToolbar } from '../../tableToolbar'
+import { Menu } from '#src/components/menu/menu'
+import { MenuPopover, MenuTrigger } from '#src/components/menu/menu-popover'
+import { Button } from '#src/components/button/button'
+import { Icon } from '#src/components/icon/icon-component'
+import { MenuItem, MenuItemGroup, MenuList } from '#src/components/menu/menu.atoms'
+
+describe('Table Container', () => {
+  test('should match snapshot', () => {
+    const { asFragment } = render(
+      <TableContainer>
+        <TableToolbar
+          description="125 Properties"
+          actions={
+            <Menu>
```

Comment:
For unit tests, I think it's fine to use fake text for any ReactNode props. Composing in actual atoms like `TableToolbar` and `Menu` will result in our subject's snapshot (in this case, `TableContainer`) capturing the implementation detail of those other components. When those other components are then changed, we'll see test failures here because the snapshot content won't match, which is not something we want.